packages/react/src/themes/teams/components/SplitButton/splitButtonStyles.ts

```diff
@@ -0,0 +1,58 @@
+import { ICSSInJSStyle } from '../../../types'
+import getBorderFocusStyles from '../../getBorderFocusStyles'
+import SplitButton from '../../../../components/SplitButton/SplitButton'
+
+const splitButtonStyles = {
+  button: ({ props: p, variables: v }): ICSSInJSStyle => {
+    return {
+      border: 0,
+      ':focus': {
+        ...(p.isFromKeyboard
+          ? {
+              '::before': { border: 0 },
+              '::after': { border: 0 },
+            }
+          : { ':active': { backgroundColor: v.backgroundColorActive } }),
+      },
+    }
+  },
+  root: ({ props: p, variables: v, theme: { siteVariables } }): ICSSInJSStyle => {
+    const { borderWidth } = siteVariables
+    const { ':focus': borderFocusStyles } = getBorderFocusStyles({
+      siteVariables,
+      isFromKeyboard: p.isFromKeyboard,
+    })
+
+    return {
+      border: `${borderWidth} solid ${v.borderColor}`,
+      borderRadius: v.borderRadius,
+      position: 'relative',
+      display: 'inline-block',
+
+      [`& .${SplitButton.slotClassNames.toggleButton}`]: {
+        borderRight: 0,
+        borderTop: 0,
+        borderBottom: 0,
+      },
+
+      ':focus-within': {
```

Comment:
We cannot use focus-within, it is not supported in IE11. Maybe extracting state prop for this would be better choice. @layershifter thoughts?

Reply:
We decided to use `:focus-within` for now. And then include polyfill or use the state value.

***

To be honest, styling overrides in this component are very complicated 😨

Reply:
it's not supported by Edge either. So we're consistent.

It's what we decided for now.